lib/maintainers.nix

```diff
@@ -1,799 +1,3946 @@
+  };
+  cstrahan = {
+    email = "charles@cstrahan.com";
+    github = "cstrahan";
+    name = "Charles Strahan";
+  };
+  cwoac = {
+    email = "oliver@codersoffortune.net";
+    github = "cwoac";
+    name = "Oliver Matthews";
+  };
+  danbst = {
+    email = "abcz2.uprola@gmail.com";
+    github = "danbst";
+    name = "Danylo Hlynskyi";
+  };
+  dancek = {
+    email = "hannu.hartikainen@gmail.com";
+    github = "dancek";
+    name = "Hannu Hartikainen";
+  };
+  danharaj = {
+    email = "dan@obsidian.systems";
+    github = "danharaj";
+    name = "Dan Haraj";
+  };
+  danielfullmer = {
+    email = "danielrf12@gmail.com";
+    github = "danielfullmer";
+    name = "Daniel Fullmer";
+  };
+  dasuxullebt = {
+    email = "christoph.senjak@googlemail.com";
+    github = "dasuxullebt";
+    name = "Christoph-Simon Senjak";
+  };
+  david50407 = {
+    email = "me@davy.tw";
+    github = "david50407";
+    name = "David Kuo";
+  };
+  davidak = {
+    email = "post@davidak.de";
+    github = "davidak";
+    name = "David Kleuker";
+  };
+  davidrusu = {
+    email = "davidrusu.me@gmail.com";
+    github = "davidrusu";
+    name = "David Rusu";
+  };
+  davorb = {
+    email = "davor@davor.se";
+    github = "davorb";
+    name = "Davor Babic";
+  };
+  dbohdan = {
+    email = "danyil.bohdan@gmail.com";
+    github = "dbohdan";
+    name = "Danyil Bohdan";
+  };
+  dbrock = {
+    email = "daniel@brockman.se";
+    github = "dbrock";
+    name = "Daniel Brockman";
+  };
+  deepfire = {
+    email = "_deepfire@feelingofgreen.ru";
+    github = "deepfire";
+    name = "Kosyrev Serge";
+  };
+  demin-dmitriy = {
+    email = "demindf@gmail.com";
+    github = "demin-dmitriy";
+    name = "Dmitriy Demin";
+  };
+  derchris = {
+    email = "derchris@me.com";
+    github = "derchris";
+    name = "Christian Gerbrandt";
+  };
+  dermetfan = {
+    email = "serverkorken@gmail.com";
+    github = "dermetfan";
+    name = "Robin Stumm";
+  };
+  desiderius = {
+    email = "didier@devroye.name";
+    github = "desiderius";
+    name = "Didier J. Devroye";
+  };
+  devhell = {
+    email = "\"^\"@regexmail.net";
+    github = "devhell";
+    name = "devhell";
+  };
+  dezgeg = {
+    email = "tuomas.tynkkynen@iki.fi";
+    github = "dezgeg";
+    name = "Tuomas Tynkkynen";
+  };
+  dfordivam = {
+    email = "dfordivam+nixpkgs@gmail.com";
+    github = "dfordivam";
+    name = "Divam";
+  };
+  dfoxfranke = {
+    email = "dfoxfranke@gmail.com";
+    github = "dfoxfranke";
+    name = "Daniel Fox Franke";
+  };
+  dgonyeo = {
+    email = "derek@gonyeo.com";
+    github = "dgonyeo";
+    name = "Derek Gonyeo";
+  };
+  dipinhora = {
+    email = "dipinhora+github@gmail.com";
+    github = "dipinhora";
+    name = "Dipin Hora";
+  };
+  disassembler = {
+    email = "disasm@gmail.com";
+    github = "disassembler";
+    name = "Samuel Leathers";
+  };
+  dizfer = {
+    email = "david@izquierdofernandez.com";
+    githu
```

Comment:
My github id is cillianderoiste, thanks!

Reply:
Thanks, fixed.